tests/template_tests/syntax_tests/test_url.py

```diff
@@ -251,3 +251,23 @@ def test_url_namespace03(self):
         context = RequestContext(request)
         output = template.render(context)
         self.assertEqual(output, '/ns2/named-client/42/')
+
+    @setup({'url-namespace04': '{% url "app:named.client" 42 %}'})
+    def test_url_namespace04(self):
```

Comment:
could you use more semantic test names or add a comment about the purpose of each test?